aurelia-auth0/auth-templates
# Skeleton-esnext-hlp-fuse

@adriatic 
You can merge this if you want

---

**Information**

Updated it to fusebox loader
Updated most, removed many parts of "build" folder since I dont need it for fusebox loader
Everything with jspm is removed.

---

**In the actual app "src" Ive done this**

> app.js

* removed step, and put it in the authservice with some changes
* also set login route title based if we are logged in

> login.js

* Removed most, and just login or logout and set title correct based on current status

> auth0-service

* location.hash to auth0 function, as set route directly before router is configured  (this is the reset bug)
* isAuthenticated is now a setter
* added a login function
* redirect uri is dynamic, so no need to edit if you export and throw on gh-pages




**How to test before merging:**
`git clone -b skeleton-esnext-hlp-fuse https://github.com/vegarringdal/auth-templates`
`cd .\auth-templates\skeleton-esnext-hlp-fuse`
`npm install`
`gulp watch`

Ive also modified `gulp export` to use correct files


---

Todo:
- fix so aurelia-dialog can be used, in a few days/next week that should be possible, cant because of a bug (could downgrade from 2.1 to 2.0, but its not a showstopper atm so Ill wait)

---


Other:
More colorful sample twist 😂 
https://vegarringdal.github.io/auth-templates/skeleton-esnext-hlp-fuse/#/


--

Plugin:
This is made, just you to create a repo for it, and all PR and we can fix/change it as we go
